templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-service
+  labels:
+    k8s-app: metadata-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: metadata-service
+  template:
+    metadata:
+      annotations:
+        fluentd.packet.net/efk: stdout+stderr
+        fluentd.packet.net/efk-index: metadata-service
+      labels:
+        k8s-app: metadata-service
+    spec:
+      containers:
+        - name: metadata-service
+          env:
+          - name: API_URL
+            value: '{{ .Values.packetAPI }}'
```

Comment:
should this maybe be `upstreamAPI` or something, instead of `packetAPI`? Since it should be more generic?

Reply:
Yeah, `upstreamAPI` sounds better. Fixed.